composer.json

```diff
@@ -104,7 +104,7 @@
         "npm-asset/slick-carousel": "^1.8",
         "oomphinc/composer-installers-extender": "^1.1 || ~2.0.0",
         "symfony/dom-crawler": "~2.8 || ~3.0 || ~4.2",
-        "ymcatwincities/openy_activity_finder": "^3.0"
+        "ymcatwincities/openy_activity_finder": "^3.0 || ^4.0"
```

Comment:
too early . Let's have 4.0, let's discuss our release plan and than add it. Sorry for confusion